### DISCUSSION TITLE
공부 진행

### DISCUSSION THREAD

**Discussion by seohee0925** [WooJungE/AI_Algorithm#2] [Q&A] [answered]
파이썬으로 AI 알고리즘에 관해 공부하셨던데, 독학으로 하신걸까요? 만약 강의나 교재를 이용하였다면 혹시 알려주실 수 있을까요?

**Comment by WooJungE** [comment, marked as answer]
학교 수업과 병행하면서 자습도 했습니다~
파이썬 알고리즘 참고했습니다!
